hydromt/utils.py

```diff
@@ -65,3 +66,13 @@ def _dict_pprint(d):
     import json
 
     return json.dumps(d, indent=2)
+
+
+def has_no_data(data) -> bool:
```

Comment:
Can you specify the type of data that can be passed here? 
xr.Dataset only and not xr.DataArray? Can pd.DataFrame, gpd.GeoDataFrame and numpy.array be passed too?

Reply:
good point! I forgot about it.